### DISCUSSION TITLE
Changes to make volume shortcuts behave like Stable

### DISCUSSION THREAD

**Discussion by Osu-user** [ppy/osu#31358] [Ideas] [answered]
This is just a small shortcut change that has been bugging me down when playing in Lazer:

1. In osu! stable you don't need to hold alt to change the volume while in the home menu or in gameplay mode, simply pressing up or down works. 
2. You need to repeatedly press the arrow keys to change volume, in stable you can simply hold. (Already fixed in #31282 )

**Comment by peppy** [comment, marked as answer]
> In osu! stable you don't need to hold alt to change the volume while in the home menu or in gameplay mode, simply pressing up or down works.

won't be returning sorry. you can manually change the binding but it will mean you can't use it when a textbox is selected, as one example.
